your_daily_dose_malware/main.py

```diff
@@ -1,12 +1,73 @@
 import click
 from commands.scraper import scraper
+import requests
+from dotenv import load_dotenv
+import os
+load_dotenv()
+@click.command(
+    help="""
+    Download eitheir hundred recent malwares uploaded within the last 60 min or  all the recent malwares uploaded within the last 48 hours sha256 hash by using api from 'https://mb-api.abuse.ch/api/v1/' 
 
+    -s256 or --by-sha256: get the lastest sha256 hashes from 'https://bazaar.abuse.ch/export/txt/sha256/recent' save them in sha256_names.txt then for each file  download all malwares in zip file
 
-@click.group()
-def scraper():
-    """:) Your Daily Dose Malware :)"""
-
+    -hr or --hundred-recent: get the lastest hundred recent malwares uploaded within the last 60 min
+    """)
+@click.option('-s256','--by-sha256',is_flag=True)
+@click.option('-hr','--hundred-recent',is_flag=True)
+def run_scrapper(by_sha256, hundred_recent):
+    headers = {
+        'API-KEY':os.getenv('API_KEY')
+    }
+    if hundred_recent:
+        response = requests.post('https://mb-api.abuse.ch/api/v1/', data={'query':'get_recent','selector':'100'}, headers=headers)
+        json_response = response.json()
+        
+        if json_response['query_status'] == 'no_selector':
+            click.echo('	No selector provided. Please use either time or limit as selector',color=True)
```

Comment:
I wouldn't do all this echos at this point. Since the tool will be using multiple backends, I would say its best to use something like rich, to display back green or red lines, plus 1 line informational if you were able to download samples from each backend.

Reply:
I will keep it in mind but if is not really important  I will just delete it

Reply:
I was even though  about to add rich but finally i have remove it to keep it simple but like i said if it's not really  importanti will add it later

Reply:
Sure we can fix this cosmetic issues later.